deeplinkdispatch/src/test/java/com/airbnb/deeplinkdispatch/DeepLinkEntryTest.java

```diff
@@ -11,13 +11,20 @@
   @Test public void testSingleParam() {
     DeepLinkEntry entry = deepLinkEntry("airbnb://foo/{bar}");
 
-    assertThat(entry.getParameters("airbnb://foo/myData").get("bar")).isEqualTo("myData");
+    final Optional<DeepLinkMatch> matches = entry.matches("airbnb://foo/myData");
```

Comment:
nit: you don't need to make these local variables final, can remove to reduce clutter